SocialFilter/sections/whitelist.txt

```diff
@@ -14,6 +14,8 @@ thehungryjpeg.com,softwarespro.net#@#.fb-like
 @@||platform-api.sharethis.com/js/sharethis.js$domain=itsmycity.ru
 ! https://github.com/AdguardTeam/AdguardFilters/issues/37719
 tiempo.com.mx#@#.socialShare
+! https://github.com/AdguardTeam/AdguardFilters/issues/39515
+carfax.com#@#.fb-button
```

Comment:
I don't see this button

Reply:
> I don't see this button

It must be unblocked because it's login via facebook.

<details><summary>no exception:</summary>

![3](https://user-images.githubusercontent.com/33488229/64062026-b589bf00-cbea-11e9-9030-a27dc943aa26.png)
</details><br/>

<details><summary>with exception:</summary>

![4](https://user-images.githubusercontent.com/33488229/64062030-bcb0cd00-cbea-11e9-9e29-1661702291a4.png)
</details><br/>